compiler/luci/service/src/Nodes/CirclePad.test.cpp

```diff
@@ -90,3 +90,56 @@ TEST(ShapeRuleTest, pad_without_padding_NEG)
 
   ASSERT_ANY_THROW(shape_inf_rule.infer(node_pad, shape));
 }
+
+TEST(ShapeRuleTest, pad_non_const_paddings)
```

Comment:
@icodo98, @seanshpark,

IMHO, it seems to me that this PR implements handling for the exception case for Non-const `padding` node.

So the case related to this implementation can be considered as a `NEG` test,
so this itself can be considered as a `NEG` test already.
What do you think?

```suggestion
TEST(ShapeRuleTest, pad_non_const_paddings_NEG)
```

Reply:
> IMHO, it seems to me that this PR implements handling for the exception case for Non-const `padding` node.

I completely agree with that opinion. 😊

Reply:
Not sure, from our SE, definition of `negative` was ambiguous to me.

From the Pad Op,
- padding can be constant
- padding can be non constant

I think these two are normal cases.
For the point of return shape, now shape can have unknown dims as normal cases.
So, as we treat non constant and return shape with all unknown dims looks normal.
Strictly speaking, I don't think this is negative test.